modules/swagger-parser-v2-converter/src/test/resources/issue-33.json

```diff
@@ -0,0 +1,68 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Examples"
+    },
+    "paths": {
+        "/users": {
+            "get": {
+                "produces": [
+                    "application/json",
+                    "text/csv"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/User"
+                            }
+                        },
+                        "examples": {
+                            "application/json": [
+                                {
+                                    "id": 5,
+                                    "name": "Ford Prefect",
+                                    "friends": [
+                                        42
+                                    ]
+                                },
+                                {
+                                    "id": 17,
+                                    "name": "Slartibartfast"
+                                }
+                            ],
+                            "text/csv": "5,Ford Prefect,5\n17,Slartibartfast\n",
+                            "application/yaml": {
+                                "foo": "bar",
+                                "random": "example"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
```

Comment:
Can you add a case to the test where one of the definitions has a property with `x-nullable`?

Reply:
Checked! Test case added!